edx/edx-ora2
# Added LaTeX preview feature to all comment/feedback fields

ORA2 already offers LaTeX previewing when LaTeX is enabled for submissions, but the peer assessment and self assessment sections also allow for inputting LaTeX, even though there are no preview buttons for these fields.

This change adds a LaTeX preview button to each of these fields that are currently missing it.

**Screenshot:**

![screenshot](https://i.imgur.com/VJ4mZY4.png)

**Testing setup:**

1. Clone this repository into the `/edx/app/src` directory, checkout the branch for this PR, and install the xblock like so:
   ```
   pip install -e file:///edx/src/edx-ora2/#egg=ora2==localdev
   ```
1. Go to the `Advanced Settings` of a course in Studio and add `"ora2"` to the *Advanced Module List*
1. Add an "Open Response Assessment" problem to a new Unit
1. Select "Edit" on the new ORA problem to open the settings modal
1. Change "Allow LaTeX Responses" to `True`, and click *Save* to close the settings dialog
1. Publish the Unit

**Testing instructions:**

1. Logged in with two different users (I'll refer to them as *User A* and *User B*), navigate to the new ORA2 problem in the LMS
1. *User A*: Type some LaTeX markup into the submission field, for example:
   ```latex
   \begin{align}
   u_{n}&amp;=-3+\frac{3n}{2}\\
   &amp;=\frac{3n-6}{2}
   \end{align}
   ```
1. *User A*: Select "Preview in LaTeX" to verify that it (still) works as expected
1. *User A*: Submit the response and then complete the "Learn to Assess Responses" section
1. *User B*: Submit a response, and then complete the "Learn to Assess Responses" section - you will be prompted to assess *User A*'s submission in the "Assess Peers" section
1. *User B*: Observe that there are "Preview in LaTeX" buttons below each of the fields
1. *User B*: Type some LaTeX markup into each of the fields and verify that the "Preview in LaTeX" feature works as expected
